src/MultiSelect/index.js

```diff
@@ -161,6 +189,15 @@ class MultiSelect extends Component<Props, State> {
       case 9: // tab
         // Blur input, close dropdown
         return;
+      case 13: // enter
+        if (inputValue !== "") {
+          this.onAddTag({
+            label: startCase(inputValue),
+            value: inputValue.toLowerCase(),
+          });
+        }
+        this.setState({ inputValue: "" });
```

Comment:
This may not be needed– the state reducer for downshift should catch that an item has been selected (after the onAddTag) and clear the inputValue for us.